src/main/java/org/terasology/customOreGen/PocketStructureDefinition.java

```diff
@@ -113,18 +115,19 @@ public DiffusePocketStructure(Transform transform, Random random, Vector3ic chun
             if (rMax < 0) {
                 rMax = 0;
             }
-            float[] bb = new float[]{-rMax, -rMax, -rMax, rMax, rMax, rMax};
-            transform.transformBB(bb);
+            Vector3f min = new Vector3f();
+            Vector3f max = new Vector3f();
+            transform.frustumAabb(min, max);
```

Comment:
You need to take `rMax` into account too. This computes the bounding box for a 2*2*2 cube. You could do this by adding `transform.scale(rMax);` before `transform.frustumAabb(min, max);` (or something that has the same effect on `min` and `max` without messing up `transform`).